src/applications/vaos/components/calendar/CalendarNavigation.jsx

```diff
@@ -12,6 +12,7 @@ const CalendarNavigation = ({
       className="vads-u-display--flex vads-u-font-weight--normal vads-u-padding--0 vads-u-color--link-default"
       onClick={prevOnClick}
       disabled={prevDisabled}
+      type="button"
```

Comment:
Without this `type`, these buttons were causing JS warnings related to form submission